src/platformio.ini

```diff
@@ -391,6 +391,24 @@ upload_flags =
 	BOOTLOADER=bootloader/ghost/ghost_tx_bootloader.bin
 	VECT_OFFSET=0x4000
 
+[env:GHOST_2400_TX_LITE_via_STLINK]
+extends = env_common_stm32
+platform = ststm32@11.0.0
+board = GHOST_TX
+build_flags =
+	${env_common_stm32.build_flags}
+	${common_env_data.build_flags_tx}
+ 	-D TARGET_TX_GHOST_LITE
```

Comment:
I wonder if you can also include -D TARGET_TX_GHOST here, then have an elif for the different power settings.  This should remove the need for all of the other TARGET_TX_GHOST_LITE additions around the code.

Reply:
Good idea! Cleans it up a bit. I did not know if I needed to tweak anything else. 

RIght now I cannot get the power to play nice... something is off. Maybe we can talk about it on discord but I might have to do some probing.